pandas/core/series.py

```diff
@@ -1231,6 +1232,14 @@ def mode(self):
         # TODO: Add option for bins like value_counts()
         return algos.mode(self)
 
+    @Appender(base._shared_docs['unique'] % _shared_doc_kwargs)
+    def unique(self):
+        result = super(Series, self).unique()
```

Comment:
ok then

add a big TODO here with the issue reference saying this should really return a dtype compat array-like in future versions